release-notes/opensearch-release-notes-1.3.17.md

```diff
@@ -0,0 +1,24 @@
+# OpenSearch and OpenSearch Dashboards 1.3.17 Release Notes
+
+## Release Details
+
+[OpenSearch and OpenSearch Dashboards 1.3.17](https://opensearch.org/versions/opensearch-1-3-17.html) includes the following bug fixes, documentation and maintenance updates.
+
+OpenSearch [Release Notes](https://github.com/opensearch-project/OpenSearch/blob/main/release-notes/opensearch-release-notes-1.3.17.md).
+
+OpenSearch Dashboards [Release Notes](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/release-notes/opensearch-dashboards.release-notes-1.3.17.md).
+
+
+## MAINTENANCE
+
+
+### Opensearch Security
+
+
+* Update security reachout email ([#4333](https://github.com/opensearch-project/security/pull/4333))
```

Comment:
I dont think this is related to the artifact, more like repo maintenance and changes.
Can remove.